CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # PowerShell Editor Services Release History
 
+## 0.12.0
+### Tuesday, April 4, 2017
+
+#### Fixes and improvements
+
+- Added [#408](https://github.com/PowerShell/PowerShellEditorServices/pull/408) -
+  Enabled debugging of untitled script files
+- Added [#405](https://github.com/PowerShell/PowerShellEditorServices/pull/405) -
+  Script column number is now reported in the top stack frame when the debugger
+  stops to aid in displaying a column indicator in Visual Studio Code
+- Fixed [#411](https://github.com/PowerShell/PowerShellEditorServices/issues/411) -
+  Commands executed internally now interrupt the integrated console's command
+  prompt
+- Fixed [#409](https://github.com/PowerShell/PowerShellEditorServices/pull/409) -
+  PowerShell session now does not crash when a breakpoint is hit outside of
+  debug mode
+
```

Comment:
I know we documented the auto variables fix in vscode-powershell but technically the fix was in PSES.  Not sure if that is worth pointing here since there is no issue to point to.  But maybe worth a quick mention?